File: OOAD/solid-principle-app/src/isp/refactor/Manager.java
package isp.refactor;

public class Manager implements IWorkableEatable{
	@Override
	public void startWork() {
		System.out.println("Manager Start Working");
		
	}

	@Override
	public void stopWork() {
		System.out.println("Manager Stop Working");
	}

	@Override
	public void startEating() {
		System.out.println("Manager Start Eating");
	}

	@Override
	public void stopEating() {
		System.out.println("Manager Stop Eating");
	}
}

File: C#/OOP/SOLID Principle/InterfaceSegregation-App/InterfaceSegregation-App/Manager.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace InterfaceSegregation_App
{
    class Manager : IWorkableEatable
    {
        public void StartEating()
        {
            Console.WriteLine("Manager Start Eating");
        }

        public void StopEating()
        {
            Console.WriteLine("Manager Stop Eating");
        }

        public void StartWorking()
        {
            Console.WriteLine("Manager Start Working");
        }

        public void StopWorking()
        {
            Console.WriteLine("Manager Stop Working");
        }
    }
}
